magenta/protobuf/music.proto

```diff
@@ -0,0 +1,125 @@
+syntax = "proto3";
+
+package tensorflow.magenta;
+
+// A message containing a symbolic music sequence. The design is largely
+// based on MIDI but it should be able to represent any music sequence.
+// For details see https://www.midi.org/specifications.
+message NoteSequence {
+  // NOTE: The first three fields are duplicated from SequenceFile to allow
+  // creation of portable SSTables of Sequence protos.
+  // Unique id. See data_utils.h GenerateSequenceId().
+  string id = 1;
+  // The path of the file relative to the root of the collection.
+  string filename = 2;
+  // The collection from which the file comes. This can be shorthand e.g.
+  // "bach". One purpose is to allow for easy selection of all or some files
+  // from a particular source.
+  string collection = 3;
```

Comment:
I was talking about "collection". Presumably, there are a finite number of interesting MIDI collections out there, but, once we're in OSS land, there's no central naming authority for them. So, I don't think they will be unique. (Unless I'm missing something?) You don't say anything about this being either unique or human readable. A comment about that would be helpful. If it's not unique, then a name that emphasizes the field's purpose would also be helpful.